### PR TITLE
Don't use insecure SSL ciphers

### PR DESCRIPTION
**Pull request by moshekaplan**
Note about the `!SHA1:!SHA256:!SHA384` at the end:

"You can use !SHA1:!SHA256:!SHA384 to disable all CBC mode ciphers. There are some non-CBC false positives that will also be disabled (RC4, NULL), but you probably also want to disable them anyway.
Note that while GCM and CHACHA20 ciphers have SHA* in their name, they're not disabled because they use their own MAC algorithm. The SHA* in their name is for the PRF, not the MAC"

https://security.stackexchange.com/questions/166484/how-to-disable-cbc-mode-ciphers

### CODE REVIEW

**Comment by B0fH** [comment]
Hi @moshekaplan -
Kindly review the comment I've posted above.

Thanks,
Elazar

**Comment by moshekaplan** [comment]
> Hi @moshekaplan - Kindly review the comment I've posted above.
> 
> Thanks, Elazar

Which comment is this? I don't see any.

**Review by B0fH — CHANGES_REQUESTED** [review]
Hi @moshekaplan -
Kindly restore the newline and then I can go ahead and merge this.
Thanks,
Elazar

File: hxtool.py [hunk 1]
```diff
@@ -725,4 +726,4 @@ def hxtool_upgrade():
 				
 else:
 	# Running under gunicorn/mod_wsgi
-	app_init(debug = False)
```
[the remark this answers is not in the document]
Comment (moshekaplan) [reply hxtool.py]: Fixed

**Pull request merged**